android/java/org/chromium/chrome/browser/BraveRewardsUserWalletActivity.java

```diff
@@ -60,10 +61,9 @@ private void SetUIControls() {
         btnGotoProvider.setText(String.format(
                 getResources().getString(R.string.user_wallet_goto_provider), walletTypeString));
 
-        if (status < BraveRewardsExternalWallet.NOT_CONNECTED
-                || status > BraveRewardsExternalWallet.PENDING) {
+        if (status < WalletStatus.NOT_CONNECTED) {
```

Comment:
Why is this not simply `status == WalletStatus.NOT_CONNECTED`?

Reply:
Good point. It's a legacy code. So I didn't touch it and we would need to remove it for rewards 2.5 anyways. IW I'll update it for this PR.

Reply:
Disregard what I said, actually, as I thought it's `status <= WalletStatus.NOT_CONNECTED`, for some reason. But then I suspect the `<` handles the default `-1` value, coming from `intent.getIntExtra(BraveRewardsExternalWallet.STATUS, -1);`.
Curious: why do you need this path?